content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -23,6 +23,22 @@ in an old version of longhorn-instance-manager until it is detached (by scaling
 reattached (by scaling its consuming workload up). Consider scaling workloads down and back up again as soon as possible
 after upgrading from a version without this mechanism (v1.5.1 or older) to v{{< current-version >}}.
 
+### Default Priority Class
+
+Longhorn v1.6.0 introduces the default Priority Class `longhorn-critical`, which has the highest value and ensures that Longhorn pods are not evicted by kube-scheduler when system resources are low.
+
+During upgrades, Longhorn applies the default Priority Class to components depending on specific settings.
+
+- When all volumes are detached and you did not specify a value for the global Priority Class setting `priority-class`, the default Priority Class is applied to all Longhorn components. `priority-class` is updated.
+- When all volumes are detached and you specified a value for the global Priority Class setting `priority-class`, the default Priority Class is applied only to user-deployed components. `priority-class` is not updated.
+- When one or more volumes are attached and you did not specify a value for `PriorityClass` in the `chart/value.yaml` or `longhorn/deploy/longhorn.yaml`, the default Priority Class is applied only to user-deployed components. `priority-class` is not updated.
+
+If you want to apply the default Priority Class to system-managed components, you must detach all volumes and change the Priority Class default setting value after the upgrade is successfully completed.
+
+You can change these behaviors by customizing the following before starting the upgrade process:
+- For user-deployed components: `defaultSetting.priorityClass` or `priorityClass` parameters in the `values.yaml` file of the [Longhorn Helm chart](https://github.com/longhorn/longhorn/blob/master/chart/)
+- Config map in [longhorn/deploy/longhorn.yaml](https://raw.githubusercontent.com/longhorn/longhorn/v1.6.0/deploy/longhorn.yaml)
+
```

Comment:
```suggestion
You can change these behaviors by customizing the following before starting the upgrade process:

- For user deployed components: `priorityClass` parameters for each component in the `values.yaml` file of the [Longhorn Helm chart](https://github.com/longhorn/longhorn/blob/v1.6.0/chart/values.yaml)
- For system managed components: `defaultSetting.priorityClass` in the `values.yaml` file of the [Longhorn Helm chart](https://github.com/longhorn/longhorn/blob/v1.6.0/chart/values.yaml)
```

For ConfigMap `longhorn-default-setting`, it's redundant, because it has been covered in `defaultSetting.priorityClass`, right?

Reply:
Yes. Just mentioned that deploying by the manifest YAML file.
Modified. Thanks.